### PR TITLE
Add Noxary Valhalla support

### PR DESCRIPTION
**Pull request by Gondolindrim**
Adds support for Noxary Valhalla.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/noxary/valhalla/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+![valhalla](https://cdn.shopify.com/s/files/1/0071/6377/3043/products/BG_AngleL_a247224d-f9ca-4093-b0de-df776e34b02d_900x.jpg?v=1598742611)
+
+An ergonomic 65% by Noxary
+
+* Keyboard Maintainer: [Gondolindrim](https://github.com/Gondolindrim)
+* Hardware Supported: proprietary PCB
+* Hardware Availability: [Noxary](https://noxary.co/collections/valhalla). As of november 17, the group buy has ended.
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/noxary/valhalla/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

File: keyboards/noxary/valhalla/rules.mk [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = STM32F072
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = full      # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/noxary/valhalla/rules.mk:10]: There is no "full" bootmagic anymore, only the "lite" option, which can be turned on with just "yes".
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/noxary/valhalla/rules.mk [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
```
Comment (drashna) [inline keyboards/noxary/valhalla/rules.mk:21]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/noxary/valhalla/readme.md (hunk 1 above)
Comment (Gondolindrim) [reply keyboards/noxary/valhalla/readme.md:11]: Committed in a3a9b0c926

File: keyboards/noxary/valhalla/rules.mk (hunk 3 above)
Comment (Gondolindrim) [reply keyboards/noxary/valhalla/rules.mk]: Both suggestions committed in 0e72a52453

File: keyboards/noxary/valhalla/readme.md (hunk 1 above)
Comment (Gondolindrim) [reply keyboards/noxary/valhalla/readme.md:24]: Committed in a3a9b0c926

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**